get-pmm.sh

```diff
@@ -179,7 +197,13 @@ install_docker() {
   if ! docker ps 1>/dev/null; then
     root_is_needed='yes'
     if ! run_root 'docker ps > /dev/null'; then
-      die "${RED}ERROR: cannot run "docker ps" command${NOFORMAT}"
+      if is_darwin; then
+        open --background -a Docker
+        echo "Giving docker desktop time to start"
+        sleep 30
+      else
+        die "${RED}ERROR: cannot run "docker ps" command${NOFORMAT}"
+      fi
```

Comment:
To be honest, I don't get the idea here...
1. As I understand `else` is related to `if` on 199 string.
2. We need to test `docker ps` after `sleep 30` again.

Reply:
At this point in the code docker is confirmed installed, but not running..I've only added a case to attempt to start docker on mac...on linux it'll just die as it did before...in the future you could certainly add more logic for monitoring the status of the start request but that wasn't the goal at this point...just handle a common use case on a workstation (program installed but not running). If nothing else I did realize I failed to encapsulate the command in run_root so I'll fix that!